randovania/games/super_metroid/logic_database/Brinstar.txt

```diff
@@ -672,15 +675,22 @@ Extra - asset_id: 92
   * Layers: default
   * Normal Door to Big Pink/Door to Green Hill Zone
   > Door to Morph Ball Room
-      Hi-Jump Boots or Space Jump or Wall Jumps (Beginner)
+      Any of the following:
+          Hi-Jump Boots or Space Jump or Wall Jumps (Beginner)
+          # IBJ
+          Vertical Infinite Bomb Jump (Intermediate) and Can Lay Bombs
   > Door to Noob Bridge
       Trivial
   > Pickup (Missile Expansion)
       All of the following:
-          Morph Ball
+          Any of the following:
+              Morph Ball
+              # Tunnel Crawl: https://youtu.be/XfTiy5JMfRE
+              Movement (Advanced)
           Any of the following:
               Hi-Jump Boots or Space Jump or Wall Jumps (Beginner)
-              Morph Ball Bombs and Morph Ball and Vertical Infinite Bomb Jump (Intermediate)
+              # IBJ
```

Comment:
cant you do this with a DIBJ too?

Reply:
Diagonal bomb jumps are not in the DB. As far as I know I've only ever seen Vertical jumps and Horizontal jumps. A combo of the the two will get you wherever you need to go. Linear Algebra strikes again.

Reply:
my bad, in am2r circles we refer to horizontal bomb jumps as diagonal bomb jumps. since there, they go more diagonally up, than fully horizontal.